Makefile

```diff
@@ -113,6 +113,10 @@ gomod-verify:
 	$(GO_CMD) mod tidy
 	git --no-pager diff --exit-code go.mod go.sum
 
+.PHONY: toc-verify
+toc-verify:
+	./hack/verify-toc.sh
+
 .PHONY: vet
```

Comment:
Should we also add `update-toc` to run the update-toc.sh.

Reply:
Done